packages/manager/apps/telecom/src/components/telecom/telephony/sidebar/telephony-sidebar.service.js

```diff
@@ -23,79 +23,93 @@ angular.module('managerApp').service('TelephonySidebar', function TelephonySideb
+              id: billingAccount.billingAccount,
+              title: billingAccount.getDisplayedName(),
+              state: 'telecom.telephony',
+              stateParams: {
+                billingAccount: billingAccount.billingAccount,
+              },
+              allowSubItems: billingAccount.services.length > 0,
+            }, self.mainSectionItem);
+
+            /* ----------  Numbers (alias) display  ---------- */
+
+            // first sort numbers of the billingAccount
+            const sortedAlias = tucVoipService
+              .constructor.sortServicesByDisplayedName(billingAccount.getAlias());
+
+            // add number subsections to billingAccount subsection
+            addServiceMenuItems(sortedAlias, {
+              state: 'telecom.telephony.alias',
+              prefix: $translate.instant('telecom_sidebar_section_telephony_number'),
+            }, billingAccountSubSection);
+
+            /* ----------  Lines display  ---------- */
+
+            // first sort lines of the billingAccount
+            const sortedLines = tucVoipService
+              .constructor.sortServicesByDisplayedName(billingAccount.getLines());
+
+            // display lines except plugAndFax and fax
+            const sortedSipLines = filter(sortedLines, line => ['plugAndFax', 'fax', 'voicefax'].indexOf(line.featureType) === -1);
+
+            // add line subsections to billingAccount subsection
+            const sipTrunkPrefix = $translate.instant('telecom_sidebar_section_telephony_trunk');
+            const sipPrefix = $translate.instant('telecom_sidebar_section_telephony_line');
+
+            addServiceMenuItems(sortedSipLines, {
+              state: 'telecom.telephony.line',
+              prefix(lineService) {
+                return lineService.isSipTrunk() ? sipTrunkPrefix : sipPrefix;
+              },
+            }, billingAccountSubSection);
+
+            // second get plugAndFax
+            const sortedPlugAndFaxLines = tucVoipService.constructor
+              .filterPlugAndFaxServices(sortedLines);
+
+            // add plugAndFax subsections to billingAccount subsection
+            addServiceMenuItems(sortedPlugAndFaxLines, {
+              state: 'telecom.telephony.line',
+              prefix: $translate.instant('telecom_sidebar_section_telephony_plug_fax'),
+            }, billingAccountSubSection);
+
+            // then get fax
+            const sortedFaxLines = tucVoipService.constructor.filterFaxServices(sortedLines);
+
+            // add fax subsections to billingAccount subsection
+            addServiceMenuItems(sortedFaxLines, {
+              state: 'telecom.telephony.fax',
+              prefix: $translate.instant('telecom_sidebar_section_telephony_fax'),
+            }, billingAccountSubSection);
+
+            /* ---------- CarrierSip Lines display ----------- */
+
+            const filteredCaerierSipLines = allCarrierSipLines
+              .flat()
```

Comment:
As in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat ? Because it's not supported by `
Edge` and we don't have a polyfill

Reply:
`_.flatten`